scripts/setup

```diff
@@ -2,6 +2,12 @@
 
 cd ./server
 yarn
+
+cd ..
+
+cd ./client
+yarn
+
 cd ..
 
```

Comment:
This totally works; but sometimes using a subshell is nicer way to get back to where you started.

Ex:
```
(cd ./server; yarn)
(cd ./client; yarn)
```

The brackets start a sub-shell, and when the shell ends you're back where you started. 

FYI only - no need to change anything.

Edit: You can even get fancy with forking your sub-shells 

```
(cd ./server; yarn) &
(cd ./client; yarn) &
wait
```

Each sub-shell is forked and then run in parallel. `wait` sits there until both commands are complete.
You can file this tip under 'stupid-bash-tricks'

Reply:
I had tried this initially, but I think I ran into an issue elsewhere which made me revert my whole PR then I forgot about this after re-implementing it.